op-program/chainconfig/chaincfg.go

```diff
@@ -13,16 +16,52 @@ func OPSepoliaChainConfig() *params.ChainConfig {
 	return mustLoadChainConfig("op-sepolia")
 }
 
+//go:embed configs/*json
+var customChainConfigFS embed.FS
+
 func RollupConfigByChainID(chainID uint64) (*rollup.Config, error) {
 	config, err := rollup.LoadOPStackRollupConfig(chainID)
+	if err == nil {
+		return config, err
+	}
+	return rollupConfigByChainID(chainID, customChainConfigFS)
+}
+
+func rollupConfigByChainID(chainID uint64, customChainFS embed.FS) (*rollup.Config, error) {
+	// Load custom rollup configs from embed FS
+	file, err := customChainFS.Open(fmt.Sprintf("configs/%d-rollup.json", chainID))
 	if err != nil {
 		return nil, fmt.Errorf("failed to get rollup config for chain ID %d: %w", chainID, err)
 	}
```

Comment:
nit: It's probably worth returning a clear error message when the chain config isn't found - getting a file not found error from an embedded file system is going to be a bit confusing.
```suggestion
	if errors.Is(err, os.ErrNotExist) {
		return nil, fmt.Errorf("no rollup config available for chain ID: %d", chainID)
	} else if err != nil {
		return nil, fmt.Errorf("failed to get rollup config for chain ID %d: %w", chainID, err)
	}
```